app/shared/fastfile_peeker.rb

```diff
@@ -53,7 +53,7 @@ def fastfile_from_repo(repo_config: nil, branch: nil, sha: nil)
     end
 
     def fastfile_from_contents_map(contents_map)
-      github_action do
+      github_action(@client) do
```

Comment:
Could we add an `attr_reader` above the constructor for `@client` and refer to that here, please?

Reply:
It seems that this was on purpose to not expose the client out of the class scope I think 🤔

Reply:
That's right, I think it's best to not expose this outside of the class as a way of messaging to people "hey, don't access this ivar out of this class".

Reply:
I'll resolve the conflict this afternoon and merge it if that's ok for you!

Reply:
🎷:goat:

Reply:
D'oh

Reply:
Ahh, no worries - I didn't know about it until I read [poodr](http://www.poodr.com/). Highly recommend reading it! It's probably my favourite ruby book! 📖